zeroconf/_dns.py

```diff
@@ -225,10 +225,18 @@ class DNSAddress(DNSRecord):
     """A DNS address record"""
 
     def __init__(
-        self, name: str, type_: int, class_: int, ttl: int, address: bytes, created: Optional[float] = None
+        self,
+        name: str,
+        type_: int,
+        class_: int,
+        ttl: int,
+        address: bytes,
+        created: Optional[float] = None,
```

Comment:
We should move created to the end to be consistent with the other records. It's now and hasn't shipped yet so it won't be a breaking change

Reply:
Should `scope_id` and `created` be made keyword-only arguments?

Reply:
Yes, I think that makes sense.